.circleci/config.yml

```diff
@@ -0,0 +1,78 @@
+version: 2.1
+
+orbs:
+  win: circleci/windows@2.2.0
+
+jobs:
+  linux_build_test:
+    environment:
+      NAILGUN_ROOT: "/home/circleci/nailgun"
+    working_directory: "/home/circleci/nailgun"
+    machine:
+      # linux VM
+      image: ubuntu-1604:201903-01
+    steps:
+      - checkout
+      - run:
+          name: Install OpenJDK8
+          command: |
+            sudo apt-get update && sudo apt-get install openjdk-8-jdk
+            sudo update-java-alternatives -s java-1.8.0-openjdk-amd64
+      - run:
+          name: Install mingw-w64
+          command: |
+            sudo apt-get update && sudo apt-get install mingw-w64
+      - run:
+          name: Install Python3.6.2
+          command: |
+            pyenv install -s 3.6.2
+            pyenv global 3.6.2 system
+      - run:
+          name: Make NG
+          command: |
+            cd ${NAILGUN_ROOT}
+            make ng
+            make ng.exe
```

Comment:
Are we actually creating an exe on a linux machine? That seems... weird.

Reply:
It is in the original TravisCI